### PR TITLE
Use hockey for feedback

### PR DESCRIPTION
**Pull request by orta**
Removes the mailing based bits and pieces from the AdminVC as Hockey handles this. http://hockeyapp.net/features/feedback/

### CODE REVIEW

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -277,6 +277,12 @@
...
 		600A734317DE3F3F00E21233 /* ArtsyAPI+DeviceTokens.m in Sources */ = {isa = PBXBuildFile; fileRef = 600A734217DE3F3F00E21233 /* ArtsyAPI+DeviceTokens.m */; };
+		600B8B3B1C4F963D00776F84 /* ARHockeyFeedbackDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 600B8B3A1C4F963D00776F84 /* ARHockeyFeedbackDelegate.swift */; };
+		600B8B401C4FCEE000776F84 /* push_notifications.md in Sources */ = {isa = PBXBuildFile; fileRef = 600B8B3C1C4FCEE000776F84 /* push_notifications.md */; };
+		600B8B411C4FCEE000776F84 /* welcome_to_eigen_beta.md in Sources */ = {isa = PBXBuildFile; fileRef = 600B8B3D1C4FCEE000776F84 /* welcome_to_eigen_beta.md */; };
+		600B8B421C4FCEE000776F84 /* notifications.md in Sources */ = {isa = PBXBuildFile; fileRef = 600B8B3E1C4FCEE000776F84 /* notifications.md */; };
+		600B8B431C4FCEE000776F84 /* thanks.md in Sources */ = {isa = PBXBuildFile; fileRef = 600B8B3F1C4FCEE000776F84 /* thanks.md */; };
+		600B8B451C4FCEEC00776F84 /* CHANGELOG.yml in Resources */ = {isa = PBXBuildFile; fileRef = 600B8B441C4FCEEC00776F84 /* CHANGELOG.yml */; };
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: spotted we had a bunch of docs not accessible in Xcode

**Comment by ArtsyMobile** [comment]
&nbsp; | 4 Messages
    ------------- | ------------
        :book: | LOC: 36
        :book: | Mod: ["Artsy.xcodeproj/project.pbxproj", "Artsy/Resources/Artsy-Bridging-Header.h", "CHANGELOG.yml", "Dangerfile"]
        :book: | Added: []
        :book: | Added: []
    

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

File: Artsy/App/ARHockeyFeedbackDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import Photos
+import ARAnalytics
+
+class ARHockeyFeedbackDelegate: NSObject {
+
+    func listenForScreenshots() {
+        let mainQueue = NSOperationQueue.mainQueue()
+        let notifications = NSNotificationCenter.defaultCenter()
+        notifications.addObserverForName(UIApplicationUserDidTakeScreenshotNotification, object: nil, queue: mainQueue) { notification in
+            ar_dispatch_after(1.5, self.showFeedbackWithRecentScreenshot)
```
Comment (alloy) [inline Artsy/App/ARHockeyFeedbackDelegate.swift]: What’s this :sparkles: number about?

File: Artsy/App/ARHockeyFeedbackDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        }
+    }
+
+    func showFeedback(image:UIImage? = nil) {
+        let hockeyProvider = ARAnalytics.providerInstanceOfClass(HockeyAppProvider.self)
```
Comment (alloy) [inline Artsy/App/ARHockeyFeedbackDelegate.swift]: Maybe we should move that code out into its own pod?

File: Artsy/App/ARHockeyFeedbackDelegate.swift (hunk 2 above)
Comment (orta) [inline Artsy/App/ARHockeyFeedbackDelegate.swift]: I took it from elsewhere, turns out you shouldn't ask instantly to get the screenshot, must take a second to update behind the scenes, I should add a comment about it

File: Artsy/App/ARHockeyFeedbackDelegate.swift (hunk 3 above)
Comment (orta) [inline Artsy/App/ARHockeyFeedbackDelegate.swift]: There's value in that, but maybe not enough to warrant the overhead for a new pod IMO

File: Artsy/App/ARHockeyFeedbackDelegate.swift (hunk 2 above)
Comment (alloy) [inline Artsy/App/ARHockeyFeedbackDelegate.swift]: Once that’s in, this PR is merge-able to me.

**Pull request merged**